_docs_v7/Physical-Definition.md

```diff
@@ -97,3 +100,58 @@ The reference values $$\rho_{ref}, T_{ref}, v_{ref}$$ equal the initial state va
 
 **Note:** The initial state is also used as boundary conditions for `MARKER_FAR`.
 
+## Turbulence Models ##
+
+| Solver | Version |
+| --- | --- |
+| `*_RANS` | 7.4.0 |
+
+This section describes how to setup turbulence models for RANS simulations. In general turbulence models are selected via the option `KIND_TURB_MODEL`, corrections/versions and parameters of the models are specified via the options listed below.
+The turbulent Pradtl number can be modified with option `PRANDTL_TURB` (the default is 0.9).
```

Comment:
```suggestion
This section describes how to setup turbulence models for RANS simulations. The turbulence model is activated using the option `KIND_SOLVER= RANS`, or `KIND_SOLVER= INC_RANS`
A turbulence model can then be selected via the option `KIND_TURB_MODEL`
Different submodels and parameters are specified via the different options listed below.
The turbulent Prandtl number can be modified with the option `PRANDTL_TURB` (the default is 0.9).
```

Reply:
The "Prandtl" remark is a bit out of place... maybe first mention all KIND_TURB_MODEL and XXX_OPTIONS and then other shared options for turbulence?